app/src/main/res/values/dimens.xml

```diff
@@ -53,6 +53,9 @@
     <dimen name="tab_strip_height">48dp</dimen>
     <dimen name="view_pager_indicator_height">12dp</dimen>
 
+    <dimen name="remote_icon_size">67dp</dimen>
+    <dimen name="remote_size">202dp</dimen>
```

Comment:
We should align stuff at 8dp, 4dp if necessary. Anyway, i think this size is always overriden isn't it?

Reply:
Why 8 or 4 and not 2?
I haven't checked in the source code but from what I understand from the theory it should indeed never match.

Reply:
It's what the guidelines recommend: https://material.io/design/layout/understanding-layout.html#usage